modules/manually-maintained-credentials-upgrade.adoc

```diff
@@ -8,20 +8,58 @@
 [id="manually-maintained-credentials-upgrade_{context}"]
 = Upgrading clusters with manually maintained credentials
 
-If credentials are added in a future release, the Cloud Credential Operator (CCO) `upgradable` status for a cluster with manually maintained credentials changes to `false`. For minor release, for example, from 4.7 to 4.8, this status prevents you from upgrading until you have addressed any updated permissions. For z-stream releases, for example, from 4.7.12 to 4.7.13, the upgrade is not blocked, but the credentials must still be updated for the new release.
+The Cloud Credential Operator (CCO) `upgradable` status for a cluster with manually maintained credentials is `false` by default.
 
-Use the *Administrator* perspective of the web console to determine if the CCO is upgradeable.
+* For minor releases, for example, from 4.7 to 4.8, this status prevents you from upgrading until you have addressed any updated permissions and annotated the `CloudCredential` resource to indicate that the permissions are updated as needed for the next version. This annotation changes the `upgradable` status to `true`.
```

Comment:
@jeana-redhat , because this function was indroduced from 4.8 version, so is it better  to change the example `from 4.7 to 4.8` to `from 4.8 to 4.9` ?

Reply:
This one is tricky, since 4.9 isn't released. The other two examples can definitely be changed to e.g. `4.8.10 to 4.8.11` or `4.8.0`, but I think I need to leave this one as 4.7->4.8 until 4.9 GA.

Reply:
Added a note with a "to-do" item for post-GA. Will make myself a task to track it. Thanks (as always0 for your very keen eye :)

Reply:
that makes sence, thanks jeana.